app/src/main/java/com/github/libretube/ui/adapters/SubscriptionGroupChannelsAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package com.github.libretube.ui.adapters
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.github.libretube.api.obj.Subscription
+import com.github.libretube.databinding.SubscriptionGroupChannelRowBinding
+import com.github.libretube.db.obj.SubscriptionGroup
+import com.github.libretube.extensions.toID
+import com.github.libretube.helpers.ImageHelper
+import com.github.libretube.ui.viewholders.SubscriptionGroupChannelRowViewHolder
+
+class SubscriptionGroupChannelsAdapter(
```

Comment:
Couldn't this be a [`ListAdapter`](https://developer.android.com/reference/androidx/recyclerview/widget/ListAdapter) instead?